resources/leiningen/new/luminus/core/env/dev/resources/logback.xml

```diff
@@ -33,8 +33,8 @@
     <logger name="io.netty" level="warn" /><% endifequal %><% if undertow-based %>
     <logger name="io.undertow.websockets.core.request" level="warn" />
     <logger name="io.undertow.request" level="warn" />
-    <logger name="io.undertow.session" level="warn" />
-    <logger name="io.undertow.request" level="warn" /><% endif %>
```

Comment:
Notices a duplicate entry for "io.undertow.request" so removed, just flagging up in case there's something I'm missing

Reply:
good catch, that was definitely not intentional :)